### ISSUE TITLE
Spiculated Mass

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10112]
**Preferred term label:**
Spiculated Mass

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C2986562
https://academic.oup.com/book/24629/chapter-abstract/187963561
**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Lobular breast carcinoma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001049
- label: Spiculated
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: PMID:32809592
- def: Applies to an abnormality with irregular, radiating, or spiky edges.
- comment: A spiculated mass is a centrally dense lesion seen on mammography with sharp lines radiating from its margin. The spicules can vary greatly in length, from a few millimeters to several centimeters. In malignant lesions, spicules represent a mixture of tumor cells and fibrosis invading the normal tissue surrounding the lesion.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10112
- synonyms:
